testing/regress/ecl/prefetch4.ecl

```diff
@@ -0,0 +1,157 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//class=file
+//nohthor        - does not execute anything in parallel
+//nothor         - actions are run as spearate child queries, and dependencies may be in separate sub   graphs
```

Comment:
typo spearate

Reply:
I will fix once your change is merged to ensure it is rerun.